examples/Demo/FluentUI.Demo.Client/Documentation/Components/Checkbox/Examples/CheckboxIndeterminate.razor

```diff
@@ -0,0 +1,18 @@
+﻿<div>
+    <FluentCheckbox @bind-Value="@value" @bind-CheckState="checkState" ShowIndeterminate="false" Label="Indeterminate with label" />
+    <br />
+    <p>Value is @(value ? "checked" : "unchecked")</p>
+    <p>CheckState is @(checkState is null ? "(null indeterminate)" : checkState.Value ? "checked" : "unchecked")</p>
+
+    <FluentButton OnClick="OnClick">Display current values in console</FluentButton>
```

Comment:
I don't see the benefit of adding a button to use the Console. The two interesting values are already updated in the web page. I'll delete this part of the example which complicates understanding